{"repository":"commons-math","prod_path":"src/main/java/org/apache/commons/math/MathException.java","test_path":"src/test/java/org/apache/commons/math/MathExceptionTest.java","prod_time":"2010-11-16 06:39:56","test_time":"2010-11-16 06:39:56","add_annotation_line":0,"add_call_line":0,"add_classname_line":1,"add_condition_line":0,"add_field_line":0,"add_import_line":1,"add_packageid_line":0,"add_parameter_line":0,"add_return_line":1,"del_annotation_line":0,"del_call_line":0,"del_classname_line":1,"del_condition_line":0,"del_field_line":0,"del_import_line":0,"del_packageid_line":0,"del_parameter_line":0,"del_return_line":0,"label":"POSITIVE","pro_Commit":"f2d9bc8f934db67a41cbce8c1c375270d43109ba","test_Commit":"f2d9bc8f934db67a41cbce8c1c375270d43109ba","true_label":null}